config/releases.exs

```diff
@@ -31,6 +31,24 @@ defmodule Environment do
       _ -> nil
     end
   end
+
+  def get_local_url(key) do
+    url = get(key)
+
+    case URI.parse(get(key)) do
+      %{host: "localhost"} ->
+        url
+
+      %{host: "127.0.0.1"} ->
+        url
+
+      %{host: nil} ->
+        url
+
+      %{host: _} ->
+        raise "Make sure that the DATABASE_URL environment variable for the tests points to a local database."
```

Comment:
The method look re-usable for any URL but the error it raises is bound to DATABASE_URL.

Reply:
Maybe explicitly rename it to `get_database_url()`, what do you think?

Reply:
I think we should go with what @simonprev and @remi proposed which is to make the error message more generic by using the `key` and `host` parameters.

Reply:
Yes!